flexget/utils/template.py

```diff
@@ -229,4 +230,17 @@ def render_from_task(template, task):
     :param task: Task to render the template from.
     :return: The rendered template text.
     """
-    return render(template, {'task': task, 'now': datetime.now()})
+    variables = {'task': task, 'now': datetime.now(), 'task_name': task.name}
+    result = render(template, variables)
+    # Only try string replacement if jinja didn't do anything
```

Comment:
We could start dropping support for string replacement format ... add warning for it?